exercises/spiral-matrix/test/Tests.hs

```diff
@@ -0,0 +1,40 @@
+{-# OPTIONS_GHC -fno-warn-type-defaults #-}
+
+import Spiral (spiral)
+
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "spiral" $ do
+
+  it "empty spiral" $
+    spiral 0 `shouldBe` []
+
+  it "trivial spiral" $
+    spiral 1 `shouldBe` [ [1] ]
+
+  it "spiral of side 2" $
```

Comment:
I used `side` because I imagined that `size` could be confused with the number of elements. Which one do you think is better?

- `spiral of size 2`
- `spiral of side 2`
- `spiral of 2x2`
- `spiral of size 2x2`

Any other suggestion?